dist/index.js

```diff
@@ -3423,6 +3423,11 @@ function getTestJdkHome(version, jdksource) {
             javaHome = process.env.JAVA_HOME;
         }
     }
+    // Window path has to be in apostrophe. e.g. ''C:/Program Files/Java/***'
+    if (IS_WINDOWS && jdksource === 'github-hosted') {
+        javaHome = javaHome.replace(/\s/g, '');
+        javaHome = javaHome.replace(/ProgramFiles/g, 'Progra~1');
```

Comment:
Should there be a space in ProgramFiles?  i.e., Program Files?

Reply:
Maybe just directly replace the string with space is simpler.